common/test/acceptance/tests/lms/test_learner_profile.py

```diff
@@ -492,6 +596,23 @@ def test_user_cannot_remove_default_image(self):
         self.assert_default_image_has_public_access(profile_page)
         self.assertFalse(profile_page.remove_link_present)
 
+    def test_eventing_after_multiple_uploads(self):
+        """
+        Scenario: An event is fired when a user with a profile image uploads another image
+
+        Given that I am on my profile page with public access
+        And I upload a new image via file uploader
+        When I upload another image via the file uploader
+        Then two upload events have been emitted
+        """
+        username, user_id = self.log_in_as_unique_user()
+        profile_page = self.visit_profile_page(username, privacy=self.PRIVACY_PUBLIC)
+        self.assert_default_image_has_public_access(profile_page)
+        profile_page.upload_file(filename='image.jpg')
+        self.assertTrue(profile_page.image_upload_success)
+        profile_page.upload_file(filename='image.jpg', wait_for_upload_button=False)
```

Comment:
@cahrens I haven't been able to figure out why this is happening either. The upload button should be visible.